flepimop/gempyor_pkg/tests/shared_cli/test_parse_config_files.py

```diff
@@ -0,0 +1,186 @@
+import yaml
```

Comment:
Surprisingly, not part of the stdlib: https://pyyaml.org/wiki/PyYAMLDocumentation

Reply:
pretty sure I didn't actually end up using it, so should be able to purge. will confirm next time at the mill.

Reply:
removed